spec/policies/submission_policy_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SubmissionPolicy do
```

Comment:
Since I changed the submission policy above, I thought it was worthy of its own full spec.